specs/APIReview_AllowDrop.md

```diff
@@ -0,0 +1,103 @@
+# Background
+Currently the drag&drop functionality is default enabled in webview2 and there is no way to disable it. Some developers may want to disbale this functionality in their applications 
+based upon their scenarios. According to such feature requirements, we add the new API to provide developers with the capability to configure the drag&drop functionality.
+
+# Description
+We add a new `AllowDrop` property in `CoreWebView2Controller`. 
+This API allows end developers to toggle the drag&drop functionality easily.
```

Comment:
Does this also block dragging data out of the WebView, or only dropping into the WebView? If it also blocks dragging out of the WebView then maybe AllowDragDrop (or AllowExternalDragDrop).

Reply:
This only blocks dropping into webview2, which means when AllowDrop is false, nothing outside webview2 can be dropped into current webview2.

Reply:
Changed to AllowExternalDrop.